updatecli/updatecli.d/pod-templates/helmfile.yaml

```diff
@@ -9,12 +21,15 @@ sources:
       repository: "docker-helmfile"
       token: "{{ requiredEnv .github.token }}"
       username: "{{ .github.username }}"
+
 conditions:
   checkDockerImage:
     name: Ensure that the image "jenkinsciinfra/helmfile:<found_version>" is published
     kind: dockerImage
+    sourceID: lastRelease
     spec:
       image: "jenkinsciinfra/helmfile"
+      ## tag come from the input source
```

Comment:
Could you remind me why it's the case here?

Reply:
- The "last version" is retrieved from a github tag/release.
- So there is a condition that checks if the Docker image which tag is the version has been published.
  - If it is, then the targets proceeds to update the Docker image 
  - Otherwise it skips the targets (case of tags published but image not yet deployed)

Reply:
What I don't understand is why it's the case for this particular repo and not the other comparable ones?